velox/core/SimpleFunctionMetadata.h

```diff
@@ -591,7 +591,8 @@ class SimpleFunctionMetadata : public ISimpleFunctionMetadata {
     builder.returnType(analysis.outputType);
     int32_t position = 0;
     for (const auto& arg : analysis.argsTypes) {
-      if (ConstantChecker::isConstant[position++]) {
+      if (ConstantChecker::getSize() > 0 &&
```

Comment:
It should always be true, just do `VELOX_CHECK_GT(ConstantChecker::getSize(), 0)`